managed/cmd/pmm-managed/main.go

```diff
@@ -683,6 +676,8 @@ func main() {
 
 	logger.SetupGlobalLogger()
 
+	time.Sleep(time.Second * 8)
```

Comment:
Why it's needed here? Can you please leave a comment in the code if it's really needed?

Reply:
thx, not indented leftover after debugging session